run.py

```diff
@@ -100,6 +103,10 @@
         from bugbug.models.assignee import AssigneeModel
 
         model_class = AssigneeModel
+    elif args.goal == "backout":
+        from bugbug.models.backout import BackoutModel
+
+        model_class = BackoutModel
 
     if args.train:
         db.download()
```

Comment:
There is a bit of duplication between this script and `scripts/trainer.py`, do we want to reduce the duplication somehow?

Reply:
Yeah, I was thinking about the same!
Filed #339.